2021-05-28 00:24:30,342 INFO    MainThread:23274 [wandb_setup.py:_flush():69] setting env: {}
2021-05-28 00:24:30,342 INFO    MainThread:23274 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-28 00:24:30,342 INFO    MainThread:23274 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_002430-3tcn1c1s/logs/debug.log
2021-05-28 00:24:30,342 INFO    MainThread:23274 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_002430-3tcn1c1s/logs/debug-internal.log
2021-05-28 00:24:30,342 INFO    MainThread:23274 [wandb_init.py:init():369] calling init triggers
2021-05-28 00:24:30,343 INFO    MainThread:23274 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 2, 'batch_size': 64, 'init_lr': 0.0018950079293650714}
2021-05-28 00:24:30,343 INFO    MainThread:23274 [wandb_init.py:init():418] starting backend
2021-05-28 00:24:30,343 INFO    MainThread:23274 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-28 00:24:30,350 INFO    MainThread:23274 [backend.py:ensure_launched():132] starting backend process...
2021-05-28 00:24:30,355 INFO    MainThread:23274 [backend.py:ensure_launched():136] started backend process with pid: 23283
2021-05-28 00:24:30,356 INFO    MainThread:23274 [wandb_init.py:init():423] backend started and connected
2021-05-28 00:24:30,357 INFO    MainThread:23274 [wandb_init.py:init():465] updated telemetry
2021-05-28 00:24:30,358 INFO    MainThread:23274 [wandb_init.py:init():484] communicating current version
2021-05-28 00:24:31,095 INFO    MainThread:23274 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-28 00:24:31,096 INFO    MainThread:23274 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-28 00:24:31,318 INFO    MainThread:23274 [wandb_init.py:init():522] starting run threads in backend
2021-05-28 00:24:31,350 INFO    MainThread:23274 [wandb_run.py:_console_start():1565] atexit reg
2021-05-28 00:24:31,351 INFO    MainThread:23274 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-28 00:24:31,351 INFO    MainThread:23274 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-28 00:24:31,353 INFO    MainThread:23274 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-28 00:24:31,353 INFO    MainThread:23274 [wandb_init.py:init():546] run started, returning control to user process
2021-05-28 00:27:25,484 INFO    MainThread:23274 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-28 00:27:25,487 INFO    MainThread:23274 [wandb_run.py:_restore():1507] restore
2021-05-28 00:27:27,493 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-28 00:27:27,778 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-28 00:27:28,374 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 3864
}

2021-05-28 00:27:28,784 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-05-28 00:27:28,886 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-05-28 00:27:28,987 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-05-28 00:27:29,088 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-05-28 00:27:29,189 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-05-28 00:27:29,290 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-05-28 00:27:29,392 INFO    MainThread:23274 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-05-28 00:27:30,484 INFO    MainThread:23274 [wandb_run.py:_show_files():1879] logging synced files
